compiler/ilgen/OMRVirtualMachineRegister.cpp

```diff
@@ -39,17 +39,17 @@ OMR::VirtualMachineRegister::VirtualMachineRegister(
          "VirtualMachineRegister for %s requires pointerToRegisterType (%s) to be a pointer",
          localName, pointerToRegisterType->getName());
 
-   _integerTypeForAdjustments = pointerToRegisterType->baseType();
-   if (_integerTypeForAdjustments->isPointer())
+   TR::IlType *registerBaseType = pointerToRegisterType->baseType();
+   if (registerBaseType->isPointer())
```

Comment:
I think these changes can stay the way they are, actually. I prefer the register know what type to use based on the type of the register rather than relying on the datatype of the adjustment value passed in.